PsfLauncher/StartProcessHelper.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+#include <windows.h>
+#include <wil\resource.h>
+
+// These two macros don't exist in RS1.  Define them here to prevent build
+// failures when building for RS1.
+#ifndef PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_DISABLE_PROCESS_TREE
+#    define PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_DISABLE_PROCESS_TREE PROCESS_CREATION_DESKTOP_APPX_OVERRIDE
+#endif
+
+#ifndef PROC_THREAD_ATTRIBUTE_DESKTOP_APP_POLICY
+#    define PROC_THREAD_ATTRIBUTE_DESKTOP_APP_POLICY PROC_THREAD_ATTRIBUTE_CHILD_PROCESS_POLICY
+#endif
+
+HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR currentDirectory, int cmdShow, bool runInVirtualEnvironment, DWORD timeout)

```

Comment:
>timeout [](start = 141, length = 7)

Since you are always passing infinite for now, we can reduce the complexity by hardcoding this. At the point we have configurable wait times, we can add this parameter.